README.md

```diff
@@ -1,44 +1,154 @@
+
+Install [Docker](https://www.docker.com) by following the instructions [here](https://www.docker.com/community-edition#/download) for your preferrerd operating system.  
+You would also need a Docker Registry. [Docker Hub](https://hub.docker.com/) lets you create one quickly.
+
+# Steps
+
+1. [Clone the repo](#1-clone-the-repo)
+2. [Create IBM Cloud services](#2-create-compose-for-mongodb-service-with-ibm-cloud)
+3. [Build your images](#3-build-your-images)
+4. [Configure Deployment files](#4-configure-deployment-files)
+5. [Deploy the application](#5-deploy-the-application)
+6. [Search for a product in Amazon](#6-search-for-a-product-in-amazon)
+
+### 1. Clone the repo
+
+```
+$ git clone https://github.com/IBM/watson-review-analyzer
+$ cd watson-review-analyzer/
+```
+
+### 2. Create IBM Cloud services
+
+Create the following service:
 
 * [**Watson Discovery**](https://console.ng.bluemix.net/catalog/services/discovery)
+* [**IBM Cloud Kubernetes**](https://console.bluemix.net/containers-kubernetes/catalog/cluster)
 
-* [**Cloudant NoSQL DB**](https://console.bluemix.net/catalog/services/cloudant-nosql-db)
+### 3. Build your images
 
+The Node.js app will be packaged into a Docker image. This will be used by the Kubernetes Cluster
 
-## 2. Installation
+```
+$ docker build -t DOCKERHUB_USERNAME/watson-review-analyzer:1.0 .
+$ docker push DOCKERHUB_USERNAME/watson-review-analyzer:1.0
+```
 
-Edit the `config.json.sample` file to include your Watson Discovery username and password. Then rename the file to `config.json`.
+### 4. Configure Deployment files
+> Make sure you have set-up `kubectl` to use your cluster.
 
-Next, install all the required dependencies for the project.
+Add your Watson Discovery credentials in `config.json.sample` and rename it to `config.json`:
 
+![Discovery Credentials](docs/discovery-credentials.png)
+
+`config.json`:
 ```
-$ npm install
+{
+  "discoveryUsername" : "YOUR_WATSON_DISCOVERY_USERNAME",
+  "discoveryPassword" : "YOUR_WATSON_DISCOVERY_PASSWORD"
+}
 ```
 
-Next, provide your Cloudant URL from your Cloudant Service Credentials as an environment variable.
+Create configmap in Kubernetes for `config.json`
 
 ```
-export CLOUDANT_URL=https://1111405-438f-4374-9b43-975e0220f3cd-bluemix:935684a91478c5f8918d3d0b2e004ef6fe825939ed2304f28ec2225fa9a952df@8bb72405-438f-4374-9b43-975e0220f3cd-bluemix.cloudant.com
+$ kubectl create configmap watson-discovery-config --from-file=config.json
 ```
-Now, you are ready to run the app!
+
+### 5. Deploy the application
+<!--Deploy in kubernetes (1) nodejs app and (2) couchdb instance. Access via external ip-->
+
+Deploy the CouchDB database. This is where the reviews and Watson Discovery results will be stored.
```

Comment:
theres a fair bit of switching between cloudant and couchdb, i get they are both nosql dbs, but we should pick one and be consistent about it, OR just say "nosql db" instead of cloudant and/or couchdb, and state which ones we've tested.